src/persist/src/sqlite.rs

```diff
@@ -208,6 +182,77 @@ impl Consensus for SqliteConsensus {
             Ok(Err(current))
         }
     }
+
+    async fn scan(
+        &self,
+        _deadline: Instant,
+        key: &str,
+        from: Option<SeqNo>,
+    ) -> Result<Vec<VersionedData>, ExternalError> {
+        let conn = self.conn.lock().await;
+        let mut stmt = conn.prepare_cached(
+            "SELECT sequence_number, data FROM consensus
+                 WHERE shard = $shard AND sequence_number >= $from
+                 ORDER BY sequence_number DESC",
+        )?;
+        let from = from.unwrap_or(SeqNo(0));
```

Comment:
I think this suggests that the API could just be SeqNo instead of Option<SeqNo>. I can't think of an impl where those would return different results

Reply:
done.